scripts/Interface/ui/drill/view/DrillView.py

```diff
@@ -130,6 +132,7 @@ def show(self):
         Override QMainWindow::show. It calls the base class method and
         updates the instrument.
         """
+        self.setWindowBehaviour()
```

Comment:
Have we not already set the window flags in the constructor, how come we have to set them again?

Reply:
see my other comment, but I think I meant to remove the call to `setWindowBehaviour` in `scripts/Drill.py` and call it here instead!